core-bundle/src/Migration/Version413/RelLightboxMigration.php

```diff
@@ -93,7 +93,9 @@ private function getTargets(): array
             }
 
             foreach ($GLOBALS['TL_DCA'][$tableName]['fields'] ?? [] as $fieldName => $fieldConfig) {
-                if (0 === strpos($fieldConfig['eval']['rte'] ?? '', 'tiny')) {
+                $rte = $fieldConfig['eval']['rte'] ?? '';
```

Comment:
```suggestion
                $rte = $fieldConfig['eval']['rte'] ?? null;
```

This way the `is_string()` check will already return false and `strpos()` will never be called for non-string values.